.github/workflows/cli-integration.yml

```diff
@@ -61,10 +61,6 @@ jobs:
         with:
           python-version: ${{ matrix.python }}
 
-      - name: Install hdf5
-        if: matrix.python == '3.9'
-        run: sudo apt-get update && sudo apt-get install -y libhdf5-dev
```

Comment:
@jwodder is this step safe to remove now that we're not testing against Python 3.9?

Reply:
If the tests pass without it, yes.